test/tag_for_hosted.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
```

Comment:
nit: why did we move this file?
It seems previous location also looks good.

Reply:
feel here is better as it's for testing now.
After Yang is OK we use uniform name, we don't need this file, all will be controlled in cloud build yaml.